Doc/using/windows.rst

```diff
@@ -18,7 +18,7 @@ Installing Python
 Unlike most Unix systems and services, Windows does not include a system
 supported installation of Python. To make Python available, the CPython team
 has compiled Windows installers (MSI packages) with every `release
-<https://www.python.org/download/releases/>`_ for many years. These installers
```

Comment:
Thanks for this since the previous link (https://www.python.org/download/releases/) contains very old releases. As a suggestion since this page is specific to Windows https://www.python.org/downloads/windows/ seems to be a better link that has all the releases along with direct links to download Windows installers.

Reply:
@tirkarthi Thanks for the feedback!  Yes, that other link is referenced further down on this same page under 'Other Platforms', so I'm not sure which is better.  There's two ways of looking at it -- if it links to `downloads/windows`, there is a cookie crumb at the top to backup one level to downloads.  But, if it links to `downloads` directly, then there's an obvious choice (the big yellow button) for the most likely version that someone would want and immediately under that button is the link to the `downloads/windows` page.  That's why I choose to link to `downloads` here, but, as I said, I'm not sure which is better.

Reply:
Got it. Thanks for the clarification. I would wait for the reviewer's feedback 👍